owncloudApp/src/main/res/values/strings.xml

```diff
@@ -620,4 +620,10 @@
     <string name="prefs_logs_summary">Logs may contain sensitive information. Sharing logs with others is sole user responsibility</string>
 
     <string name="synchronizing_account">Synchronizing account</string>
+
+    <!-- MDM configurations -->
+    <string name="server_url_configuration_title">Server URL</string>
+    <string name="server_url_configuration_description">The server URL that will appear in the login screen by default</string>
+    <string name="server_url_input_visibility_configuration_title">Server URL visibility</string>
+    <string name="server_url_input_visibility_configuration_description">Indicates whether the input field for the server URL in the login screen is visible or not</string>
```

Comment:
Can those strings be shown within the app? Do we really need to add them to translatable strings?

Reply:
No, they aren't shown in the app, they're only kind of meta-data for MDM managed configurations, so I will move them to `strings__not_to_translate.xml`.

Reply:
They aren't shown in the app, but documentation about managed configurations say that this should be in localized resources: https://developer.android.com/work/managed-configurations#define-configuration